todo_app/app.py

```diff
@@ -71,26 +70,34 @@ def add_item():
     item = request.form.get('item') 
     #Add item to list to do
     add_item_url = "https://api.trello.com/1/cards"
+    add_item_query = buildAddItemParams(item)
+    add_item_resp = query_trello("POST", add_item_url, add_item_query)
+    return redirect(url_for('index'))
+
+def buildAddItemParams(item):
     add_item_query = trello_key_params
     add_item_query['idList'] = trello_todo_list_id
     add_item_query['name'] = item
-    add_item_resp = query_trello("POST", add_item_url, trello_key_params)
-    return redirect(url_for('index'))
+    return add_item_query
 
 @app.route('/createTodoList', methods=['GET'])
 def createTodoList():
     new_list_url = "https://api.trello.com/1/lists"
-    new_list_query_params = trello_key_params
-    new_list_query_params['name'] = "To Do"
-    new_list_query_params['idBoard'] = trello_board_id
+    new_list_query_params = buildNewTodoListParams()
     new_list_resp = query_trello("POST", new_list_url, new_list_query_params)
     if 200 != new_list_resp.status_code:
         return render_template('error.html')
     global trello_todo_list_id
-    resp_dict = json.loads(new_list_resp.text)
+    resp_dict = new_list_resp.json()
     trello_todo_list_id = resp_dict['id']
     return redirect(url_for('index'))
 
+def buildNewTodoListParams():
+    new_list_query_params = trello_key_params
```

Comment:
See the comment above
```suggestion
    new_list_query_params = trello_key_params.copy()
```